wercker.yml

```diff
@@ -1,34 +1,18 @@
-box: blisteringherb/scholarship@0.0.8
+box: blisteringherb/ds-wercker-box:mysql
```

Comment:
That seems like a good best practice to evaluate, but yeah shouldn't block this.